Rust/simple-tcp-client/Cargo.toml

```diff
@@ -0,0 +1,4 @@
+[package]
+name = "tcpclient"
+version = "0.0.1"
```

Comment:
Please use `0.1.0` for consistency with other examples

Reply:
Crate name and version has been changed now.